### DISCUSSION TITLE
Should we add non-alphabetic "words" in our vocab list for Assignment 1?

### DISCUSSION THREAD

**Discussion by iJinjin** [seominjoon/kaist-ai605#2] [Q&A] [answered]
Hello professor,

I have a question on our first assignment:
When we create the vocabulary for the training data of SST-2, should we only include "words" that consist of at least one alphabet or should we take into account words made entirely symbols such as punctuations and empty strings?

Thank you in advance!

**Comment by seominjoon** [comment, marked as answer]
This is a rather design choice (than there is a correct answer), but note that there are not that many possible punctuations, so including them won't hurt.

**Reply by iJinjin** [reply]
Thank you!
